src/views/migrationpolicies/utils/utils.ts

```diff
@@ -0,0 +1,19 @@
+import { TFunction } from 'react-i18next';
+
+import { NO_DATA_DASH } from '@kubevirt-utils/resources/vm/utils/constants';
+import { readableSizeUnit } from '@kubevirt-utils/utils/units';
+
+export const getBooleanText = (value: boolean, t: TFunction): string =>
+  value ? t('YES') : t('NO');
+
+export const getBandwidthPerMigrationText = (
+  bandwidth: string | number,
+  t: TFunction,
+): string | number => {
+  if (!bandwidth || bandwidth === '0') return t('Unlimited');
+  if (typeof bandwidth === 'string') return readableSizeUnit(bandwidth);
+  return bandwidth;
+};
```

Comment:
can you please try to see if we can import "t" from i18n instead of passing it as an arg?

Reply:
done, thanks for the tip :)